contracts/zero-ex/tests/Multiplex.t.sol

```diff
@@ -0,0 +1,1453 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+
+  Copyright 2022 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.6;
+pragma experimental ABIEncoderV2;
+
+import "forge-std/Test.sol";
+import "src/external/IFlashWallet.sol";
+import "src/features/libs/LibSignature.sol";
+import "src/features/libs/LibNativeOrder.sol";
+import "src/features/interfaces/IMultiplexFeature.sol";
+import "src/features/native_orders/NativeOrdersInfo.sol";
+import "src/features/multiplex/MultiplexFeature.sol";
+import {ForkUtils} from "utils/ForkUtils.sol";
+import "utils/TestUtils.sol";
+import "utils/DeployZeroEx.sol";
+import "../contracts/test/TestMintTokenERC20Transformer.sol";
+import "../contracts/test/tokens/TestMintableERC20Token.sol";
+import "../contracts/test/integration/TestUniswapV2Factory.sol";
+import "../contracts/test/integration/TestUniswapV2Pool.sol";
+import "../contracts/test/integration/TestUniswapV3Factory.sol";
+import "../contracts/test/integration/TestUniswapV3Pool.sol";
+import "../contracts/test/integration/TestLiquidityProvider.sol";
+
+import "@0x/contracts-erc20/contracts/src/v06/WETH9V06.sol";
+
+contract Multiplex is Test, ForkUtils, TestUtils {
+    uint256 private constant MAX_UINT256 = type(uint256).max;
+    uint256 private constant HIGH_BIT = 2 ** 255;
```

Comment:
This only gets used once. It shouldn't be broken out as a named constant

Reply:
Oh yea definitely, good point